src/main/java/Duke.java

```diff
@@ -1,10 +1,74 @@
+import java.util.Scanner;
+
 public class Duke {
+    static String horizontal = " __________________________________________________\n";
+    static String introduction = horizontal + "   Hello! I'm Duke\n" + "   What can I do for you?\n" + horizontal;
+    static String farewell = horizontal + "   Bye. Hope to see you again soon!\n" + horizontal;
+    static Task[] taskList = new Task[100];
+    static int taskCount = 0;
+    static int doneIndex = 0;
+
+    public static void echoCommandList(Task[] tasklist) {
+        System.out.println(horizontal + "Here are the tasks in your list:\n");
+        for (int i = 0; i < taskCount; i++) {
+            System.out.println("    " + (i+1) + ". [" + taskList[i].getTaskType() + "]" + "[" + taskList[i].getStatusIcon() + "] " + taskList[i].getDescription());
+        }
+        System.out.println(horizontal);
+    }
+
+    public static void echoCommandDone(String line) {
+        System.out.println(horizontal + "Nice! I\'ve marked this task as done:\n" + "   [" + taskList[doneIndex-1].getTaskType() + "][X] " + line + "\n" + horizontal);
+    }
+
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println(introduction);
+
+        String line;
+        String[] splicedLine;
```

Comment:
Would it be better to name this `String` array as `splicedInputs`, or another name with a plural connotation?
```suggestion
        String[] splicedInputs;
```